en_US/LC_MESSAGES/django.po

```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
-msgstr "Post a reply"
+msgid "view"
+msgid_plural "views"
+msgstr[0] "Review"
+msgstr[1] "Review"
 
-#: kitsune/questions/jinja2/questions/question_details.html:102
+#: kitsune/questions/jinja2/questions/question_details.html:121
 #, python-format
 msgid "Marked as spam by %(marked_by)s at %(marked_time)s"
 msgstr "Marked as spam by %(marked_by)s at %(marked_time)s"
 
-#: kitsune/questions/jinja2/questions/includes/answer.html:23 kitsune/questions/jinja2/questions/question_details.html:120
-#, fuzzy
-msgid "Posted"
-msgstr "Post"
-
-#: kitsune/questions/jinja2/questions/question_details.html:131
+#: kitsune/questions/jinja2/questions/question_details.html:162
 msgid "Attached screenshots"
 msgstr "Attached screenshots"
 
-#: kitsune/questions/jinja2/questions/includes/answer.html:57 kitsune/questions/jinja2/questions/question_details.html:142
+#: kitsune/questions/jinja2/questions/includes/answer.html:106 kitsune/questions/jinja2/questions/question_details.html:173
 #, fuzzy
 msgid "Modified {datetime} by {name}"
 msgstr "Modified by {name} on {datetime}"
 
-#: kitsune/questions/jinja2/questions/includes/answer.html:31 kitsune/questions/jinja2/questions/question_details.html:149
+#: kitsune/questions/jinja2/questions/includes/answer.html:75 kitsune/questions/jinja2/questions/question_details.html:180
 #, fuzzy
 msgid "Solution chosen by {user}"
 msgstr "Collections"
 
-#: kitsune/questions/jinja2/questions/question_details.html:149
+#: kitsune/questions/jinja2/questions/question_details.html:180
 #, fuzzy
 msgid "Chosen solution"
 msgstr "Collections"
 
-#: kitsune/questions/jinja2/questions/question_details.html:152
+#: kitsune/questions/jinja2/questions/question_details.html:187
 msgid "Read this answer in context"
 msgstr "Read this answer in context"
 
-#: kitsune/questions/jinja2/questions/mobile/question_details.html:38 kitsune/questions/jinja2/questions/question_details.html:164 kitsune/questions/jinja2/questions/question_details.html:507
+#: kitsune/questions/jinja2/questions/question_details.html:199 kitsune/questions/jinja2/questions/question_details.html:272
 #, fuzzy
 msgid "I have this problem, too"
 msgstr "1 person has this problem"
 
-#: kitsune/questions/jinja2/questions/question_details.html:176
+#: kitsune/questions/jinja2/questions/question_details.html:208
+#, fuzzy
+msgid "All Replies"
+msgstr "Replies"
+
+#: kitsune/questions/jinja2/questions/question_details.html:220
+#, python-format
+msgid ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
+msgstr ""
+"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
+"\">contributors forum</a> requesting that you be white-listed."
```

Comment:
The last part of the sentence is grammatically in correct. `If ....", there should be a second part of this sentence.